js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,7 +1,7 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js').instance;
-var ResourceLoadTracker = require('./ResourceLoadTracker').instance;
+var importedExtensionStore = require('./ExtensionStore.js').instance;
+var importedResourceLoadTracker = require('./ResourceLoadTracker').instance;
```

Comment:
> ... conversely I feel ill every time I see things instantiated in JS modules and exported essentially as static references.

In what code is anyone using `js-modules` directly? Just curious.

> At least this way, you can get a reference to the class for testing/etc. All ears for a better solution that doesn't look like worst practice Java code from 1995.

Everything I see in `ExtensionStore` is effectively static, so I don't get why most of the functions on it are not static and so used ala `ExtensionStore.get(blah)`, as opposed to `ExtensionStore.instance.get(blah)`. They can easily be duck-typed/mocked for testing ... couldn't be easier than with JavaScript (and there are libs for helping).

Anyway ... prob a topic for another day.